sqalx.go

```diff
@@ -30,6 +30,10 @@ type Node interface {
 	Close() error
 	// Begin a new transaction.
 	Beginx() (Node, error)
+	// Begin a new transaction using the provided context and options.
+	// Note that the provided parameters are only used when opening the first transaction,
+	// not on subsequent ones.
```

Comment:
🌷 Maybe this helps clarifying that the parameters are only taken into account in the outermost tx, but subsequent outer tx would still take them into account again (after the last outer one has been committed for example)
```suggestion
	// not on nested ones.
```

Reply:
Good point, I updated the doc to

```
	// Note that the provided parameters are only used when opening a new transaction,
	// not on nested ones.
```